### DISCUSSION TITLE
Show dates when labels were removed

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2820] [Mobile]
It is important that a date be shown, so that people know when Janice removed the label. In fact she removed a label last week and again this week so people will want to know if this was the first time she removed label a second time and without a date they won't be able to tell. No need to refer to where I took the snapshot from.
 That is not important.

![Screenshot_20201202-055953](https://user-images.githubusercontent.com/1230959/100813307-089b9300-347a-11eb-90f5-03c6feef655f.jpg)

**Comment by candyho** [comment]
Agree that it would be more useful with the timestamps on issues interactions. I've created an issue in our internal repository to keep track of this request. Thanks for flagging.
